pkg/metrics/metrics.go

```diff
@@ -127,3 +149,13 @@ func registerMetrics() error {
 func GetRoundTripper() http.RoundTripper {
 	return roundTripper
 }
+
+// RegisterCustomResource increases the count of the custom_resource_total metric
+func RegisterCustomResource(resourceType string) {
```

Comment:
Can this be named IncCustomResource since "Register" has it's own meaning in the prometheus client?

Reply:
@awgreene I like to think of the abstraction seen outside this particular package and how the name affects readability. 
For example, from the CSC/Opsrc controller, I `metrics.RegisterCustomResource`, which reads as `Register this custom resource with "metrics"`.
`metrics.IncCustomResource` when being read outside this package seems too technical (vs readable like an English sentence), and something that the reader might have to go into to understand the purpose of the function.

Reply:
I don't agree that you should consider cases of "being read outside this package" because Go code is shared in context of the package being imported.

My point was that Prometheus includes a "Register" function in it's package which the `metrics` package makes use of. However - this is a pretty mute point in a soon to be deprecated API so I'm not going to hold up the PR on this.